backend/index.js

```diff
@@ -24,6 +26,8 @@ app.use(cors());
 
 app.use(express.json());
 
+app.use(express.static(path.join('../frontend/build')));
```

Comment:
Use the env variable here instead of the string, and you can probably remove the `path.join(` as all the path.join does is connecting the strings in a smart way you gave him.